tests/br_table_partition/run.sh

```diff
@@ -19,7 +19,7 @@ TABLE="usertable"
 TABLE_COUNT=16
 PATH="tests/$TEST_NAME:bin:$PATH"
 
-run_sql "set @@global.tidb_enable_table_partition ='nightly'"
+run_sql "set @@global.tidb_enable_table_partition ='nightly'" || true
```

Comment:
```suggestion
run_sql "set @@global.tidb_enable_list_partition = ON;"
```

Reply:
`tidb_enable_list_partition` is only support in session scope. Must put it in each run_sql so that it can take effect.

```
mysql> set @@global.tidb_enable_list_partition = 'ON';
ERROR 1105 (HY000): Variable 'tidb_enable_list_partition' is a SESSION variable and can't be used with SET GLOBAL
```